ci-operator/step-registry/ipi/install/loki/ipi-install-loki-commands.sh

```diff
@@ -108,43 +113,43 @@ stringData:
   loki.yaml: |-
     auth_enabled: false
     chunk_store_config:
-    max_look_back_period: 0
+      max_look_back_period: 0
     ingester:
-    chunk_block_size: 1572864
-    chunk_encoding: lz4
-    chunk_idle_period: 3m
-    chunk_retain_period: 1m
-    lifecycler:
-      ring:
+      chunk_block_size: 1572864
```

Comment:
It'd rather have this in the previous commits where you convert JSON into YAML

Reply:
I'll squash these two commits - neither can be applied by CVO separately anyway